src/settings/window_geometry.rs

```diff
@@ -1,16 +1,64 @@
-#[derive(Debug, Clone)]
+use crate::renderer::Renderer;
+use crate::settings::SETTINGS;
+use crate::window::WindowSettings;
+use glutin::dpi::PhysicalSize;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct WindowGeometry {
     pub width: u64,
     pub height: u64,
 }
 
+#[cfg(unix)]
+const SETTINGS_PATH_EXTENSION: &str = ".local/share/nvim/neovide-settings.json";
```

Comment:
This is not "extension", it's just settings_path.

```suggestion
const SETTINGS_PATH: &str = ".local/share/nvim/neovide-settings.json";
```

Reply:
Hmm sorta? I already have a variable called `settings_path` defined in
https://github.com/Kethku/neovide/pull/816/files#diff-ccd3e532d23ff949faa2bcac192d9441254cd1dbeedb5aebff2214f824f8def0R19-R23

which appends the above string to the home directory to create the absolute path.

Reply:
I think that's fine.

Reply:
(i mean, having local settings_path variable and global SETTINGS_PATH with same name, so, change it)

Reply:
ohh ok. Yea, sure let me do that.